### PR TITLE
Refactored Questions logic and redering

### PR DESCRIPTION
**Pull request by robinpiets**
Mayor refactor of the `Questions` logic.

Please make sure:
- [x] to add a label
- [x] to add one or more reviewers
- [x] you followed our checklist for [functional testing](https://github.com/Amsterdam/vergunningcheck/blob/master/TESTING.md)
- [x] you added the necessary documentation (either in the markdown files or inline)
- [x] you added the necessary automated tests

### CODE REVIEW

File: packages/imtr-client/src/models/checker.ts [hunk 1]
```diff
@@ -142,31 +139,102 @@ export default class Checker {
...
+    this.permits.forEach((permit: Permit) => {
+      const outcomeDecision = permit.getDecisionById("dummy");
+
+      if (outcomeDecision) {
+        const imtrOutcome = outcomeDecision.getOutput();
+
+        let outcomeType = ClientOutcomes.PERMIT_FREE;
+
+        if (imtrOutcome === outcomes.NEED_CONTACT) {
+          outcomeType = ClientOutcomes.NEED_CONTACT;
```
Comment (svenjens) [inline packages/imtr-client/src/models/checker.ts:224]: This needs a comment :-)

**Review by svenjens — APPROVED** [review]
Great work! 🙌

File: packages/imtr-client/src/models/checker.ts (hunk 1 above)
Comment (robinpiets) [reply packages/imtr-client/src/models/checker.ts:224]: Done

**Review by afjlambert — APPROVED** [review]

**Comment by afjlambert** [comment]
LGTM!

**Pull request merged**